ui/src/app/components/prepare-migration/prepare-migration.component.ts

```diff
@@ -581,7 +628,8 @@ export class PrepareMigrationComponent implements OnInit {
 
   subscribeMigrationProgress() {
     var displayStreamingMsg = false
-    this.subscription = interval(5000).subscribe((x) => {
+    var displayDataprocMsg = false
+    this.subscription = interval(5000).subscribe((x => {
```

Comment:
Nit: convert 5000 to a constant.

Reply:
Done,
Commit: https://github.com/cloudspannerecosystem/harbourbridge/pull/519/commits/f01e38d99737aab4c0fb31aa04696b24801ae545